saffron/src/lib.rs

```diff
@@ -1560,43 +1495,40 @@ impl Iterator for CronTimesIter {
     type Item = DateTime<Utc>;
 
     fn next(&mut self) -> Option<Self::Item> {
-        match self.current {
-            IterTime::Start(time) => {
-                self.current = IterTime::Next(time);
-                if self.cron.contains(time) {
-                    Some(time)
-                } else {
-                    self.next() // call back in and get the next item
-                }
+        if let Some((front, back)) = self.bounds {
```

Comment:
Is there a semantic difference between (front, back) and (start, end)? I'm used to seeing start/end in this codebase, so seeing different names front/back confused me a little bit. If there's a meaningful difference, maybe explain it in a comment? And if there's not, maybe stick with start/end naming?

Reply:
I used `(front, back)` because when I started these changes I originally was going to include support for `DoubleEndedIterator` which uses the term `next_back`. But I decided to leave that for later so the naming kinda stuck.

Reply:
Cool. I'd suggest sticking with a consistent naming convention, either start/bounds or front/back everywhere.